app/Http/Controllers/Bible/BibleFilesetsDownloadController.php

```diff
@@ -0,0 +1,190 @@
+<?php
+
+namespace App\Http\Controllers\Bible;
+
+use App\Traits\AccessControlAPI;
+use App\Traits\CallsBucketsTrait;
+use App\Traits\BibleFileSetsTrait;
+use App\Http\Controllers\APIController;
+use App\Models\Bible\BibleFileset;
+use App\Models\Bible\Book;
+use App\Models\Bible\BibleFilesetLookup;
+use App\Transformers\BibleFileSetsDownloadTransFormer;
+
+class BibleFilesetsDownloadController extends APIController
+{
+    use AccessControlAPI;
+    use CallsBucketsTrait;
+    use BibleFileSetsTrait;
+
+    /**
+     *
+     * @OA\Get(
+     *     path="download/{fileset_id}/{book}/{chapter}",
+     *     tags={"Bibles"},
+     *     summary="Download specific fileset",
+     *     description="For a given fileset return content (text, audio or video)",
+     *     operationId="v4_internal_bible_filesets.index",
```

Comment:
@vichugofsl please rename this operationId to v4_download

Reply:
@bradflood It has been removed